AutoMaxLairUI - Code/Properties/PublishProfiles/FolderProfile.pubxml.user

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
that file should be deleted from the depot

Reply:
deleted!